examples/platform/silabs/SiWx917/SiWx917/hal/rsi_hal_mcu_m4.c

```diff
@@ -17,15 +17,11 @@
 
 #include "rsi_board.h"
 #include "rsi_chip.h"
-#include "rsi_driver.h"
```

Comment:
All of these files should be move to the `matter_support` repo

Reply:
moved